.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.pythonPath": "/usr/local/anaconda3/envs/hackrd/bin/python",
+    "python.linting.enabled": true,
+    "python.linting.pylintEnabled": false,
+    "python.linting.flake8Enabled": true,
+    "python.linting.pylamaEnabled": false
+}
```

Comment:
I'll add settings.json to .gitignore

I'm using something called black for linting

Reply:
black is one way of doing it, we can also look into some other options like pylint. We should figure out which linting tool will be the best for our needs.

Reply:
im using black right now but lmk if you find anything better